codespell_lib/data/dictionary.txt

```diff
@@ -29710,6 +29710,9 @@ promted->prompted, promoted,
 promter->prompter, promoter,
 promters->prompters, promoters,
 promting->prompting, promoting,
+promtp->prompt
+promtped->prompted
```

Comment:
We usually add all verb forms:
`promtping->prompting`

Reply:
Amended to add. :+1: